cmake/pcl_find_gl.cmake

```diff
@@ -7,7 +7,7 @@ if(POLICY CMP0072)
   cmake_policy(SET CMP0072 NEW)
 endif()
 
-find_package(OpenGL QUIET REQUIRED)
+find_package(OpenGL QUIET COMPONENTS OpenGL EGL)
 
 if(APPLE AND OPENGL_FOUND)
   if("${OPENGL_INCLUDE_DIR}" MATCHES "\\.framework")
```

Comment:
```suggestion
  if("${OPENGL_INCLUDE_DIR}" MATCHES ".framework")
```

Reply:
Probably also the same at line 21.

Reply:
The location of the headers are: /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenGL.framework

Reply:
Why do you want to include that change with this PR? I didn't touch that part.

I am also on Ubuntu (Linux) where headers are simply in `/usr/include/`, so this is entirely irrelevant.

Reply:
Well, it was just what seemed wrong since the MacOS CI's are failing.

Reply:
I am not really sure why my commit changes the behaviour of where the OpenGL headers can be found.

I am just going to commit your suggested change and see if it solves the CI build issue. But I have no way to test macOS builds locally.

Reply:
Well, it didn't resolve the issue on the CI? Did you manage to compile it locally?